### PR TITLE
Session UI

### LINKED ISSUE #62: List sessions on 2nd column of UI

### PR DESCRIPTION
**Pull request by chwoozy**
Add session panel, where session list will populate based on changes to student list (add, remove).

Close session list UI #62 

Todo:
Populates session list on changes to any students' session list.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeleteStudentCommand.java [hunk 1]
```diff
@@ -19,8 +19,8 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Deletes the student identified by the index number used in the displayed student list.\n"
-            + "Parameters: INDEX (must be a positive integer)\n"
-            + "Example: " + COMMAND_WORD + " 1";
+            + "\nParameters: INDEX (must be a positive integer)"
```
Comment (enhao25) [inline src/main/java/seedu/address/logic/commands/DeleteStudentCommand.java:22]: Hi, just want to check as the previous line ": Deletes the student identified by the index number used in the displayed student list.\n" already has an \n, does having an additional \nParameters here cause anything weird on the UI? Might be better to just keep one.

File: src/test/java/seedu/address/model/student/StudentTest.java [hunk 2]
```diff
@@ -75,5 +81,10 @@ public void equals() {
         // different address -> returns false
         editedAlice = new StudentBuilder(ALICE).withAddress(VALID_ADDRESS_BOB).build();
         assertFalse(ALICE.equals(editedAlice));
+
+        editedAlice = new StudentBuilder(BOB)
```
Comment (enhao25) [inline src/test/java/seedu/address/model/student/StudentTest.java]: Hi, just checking if this is something extra that you did that isn't really related to the UI. Can I check if the purpose of this code is to check if the equals method works as expected?

**Review by enhao25 — COMMENTED** [review]
LGTM, but has some comments on certain parts of the code.

File: src/test/java/seedu/address/model/student/StudentTest.java (hunk 2 above)
Comment (chwoozy) [reply src/test/java/seedu/address/model/student/StudentTest.java]: Yes, I wanted to check if the same student with different list would assert to be equals, however I had forgotten to change `BOB` back to `Alice` nice spot!

File: src/main/java/seedu/address/logic/commands/DeleteStudentCommand.java (hunk 1 above)
Comment (chwoozy) [reply src/main/java/seedu/address/logic/commands/DeleteStudentCommand.java:22]: Yes you are right, I will resolve in the next commit!

File: src/main/java/seedu/address/model/AddressBook.java [hunk 3]
```diff
@@ -106,7 +106,8 @@ public void removeStudent(Student key) {
     public void addSession(Name name, Session session) {
         requireAllNonNull(session, name);
         Student student = students.getStudentWithName(name);
-        student.addSession(session);
+        // student.addSession(session);
```
Comment (samleewy) [inline src/main/java/seedu/address/model/AddressBook.java]: think you forgot to remove the comment right here

**Review by samleewy — COMMENTED** [review]
just looking thru, left one comment!

File: src/main/java/seedu/address/model/AddressBook.java (hunk 3 above)
Comment (chwoozy) [reply src/main/java/seedu/address/model/AddressBook.java]: ohmilorde thanku

**Review by enhao25 — APPROVED** [review]
Yeap! LGTM

File: src/main/resources/view/SessionListCard.fxml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+      <HBox alignment="CENTER_LEFT" spacing="5">
+        <Label fx:id="id" styleClass="cell_big_label">
+          <minWidth>
+            <!-- Ensures that the label text is never truncated -->
+            <Region fx:constant="USE_PREF_SIZE" />
+          </minWidth>
+        </Label>
+<!--        <Label fx:id="name" styleClass="cell_big_label" text="\$first" wrapText="true" />-->
+      </HBox>
+      <!-- <FlowPane fx:id="tags" />-->
```
Comment (samleewy) [inline src/main/resources/view/SessionListCard.fxml]: Are these commented codes rough work? Might be better to remove.

**Review by samleewy — COMMENTED** [review]
Should be fine except the minor commented out codes in fxml. We can do another review when I merge!

File: src/main/resources/view/SessionListCard.fxml (hunk 4 above)
Comment (chwoozy) [reply src/main/resources/view/SessionListCard.fxml]: Placeholder for next iteration where we add student names into session, will remove for now.

**Review by samleewy — APPROVED** [review]
LETSGEDDIT 📈

**Pull request merged**